pkg/apis/crd/v1alpha1/types.go

```diff
@@ -427,3 +354,132 @@ type BGPPeer struct {
 	// a restart before deleting stale routes. The range of the value is from 1 to 3600, and the default value is 120.
 	GracefulRestartTimeSeconds *int32 `json:"gracefulRestartTimeSeconds,omitempty"`
 }
+
+// Source describes the source spec of the packetcapture.
+type Source struct {
+	// Namespace is the source namespace.
+	Namespace string `json:"namespace"`
+	// Pod is the source pod.
+	Pod string `json:"pod"`
+	// IP is the source IPv4 or IPv6 address.
+	IP *string `json:"ip,omitempty"`
+}
+
+// Destination describes the destination spec of the packetcapture.
+type Destination struct {
+	// Namespace is the destination namespace.
+	Namespace string `json:"namespace"`
+	// Pod is the destination pod, exclusive with destination service.
+	Pod string `json:"pod"`
+	// Service is the destination service, exclusive with destination pod.
+	Service *string `json:"service,omitempty"`
+	// IP is the destination IPv4 or IPv6 address.
+	IP *string `json:"ip,omitempty"`
+}
+
+// IPv6Header describes spec of an IPv6 header.
+type IPv6Header struct {
+	// NextHeader is the IPv6 protocol.
+	NextHeader *int32 `json:"nextHeader,omitempty" yaml:"nextHeader,omitempty"`
+	// HopLimit is the IPv6 Hop Limit.
+	HopLimit *int32 `json:"hopLimit,omitempty" yaml:"hopLimit,omitempty"`
```

Comment:
not sure if this is a useful filter for packet capture?

Reply:
no, will remove this.